core/src/test/scala/unit/kafka/server/AbstractCreateTopicsRequestTest.scala

```diff
@@ -162,7 +162,7 @@ abstract class AbstractCreateTopicsRequestTest extends BaseRequestTest {
   protected def validateErrorCreateTopicsRequests(request: CreateTopicsRequest,
                                                   expectedResponse: Map[String, ApiError],
                                                   checkErrorMessage: Boolean = true): Unit = {
-    val response = sendCreateTopicRequest(request, createTopicsSocketServer)
```

Comment:
Since we have replaced `createTopicsSocketServer` with `adminSocketServer`, can we remove the unused `createTopicsSocketServer` method?

Reply:
good point. I removed it.